alacritty_terminal/src/clipboard.rs

```diff
@@ -18,35 +18,47 @@ use std::ffi::c_void;
 use log::{debug, warn};
 
 use copypasta::nop_clipboard::NopClipboardContext;
-#[cfg(not(any(target_os = "macos", target_os = "windows")))]
+#[cfg(all(not(any(target_os = "macos", windows)), feature = "wayland"))]
 use copypasta::wayland_clipboard;
-#[cfg(not(any(target_os = "macos", target_os = "windows")))]
+#[cfg(all(not(any(target_os = "macos", windows)), feature = "x11"))]
 use copypasta::x11_clipboard::{Primary as X11SelectionClipboard, X11ClipboardContext};
-use copypasta::{ClipboardContext, ClipboardProvider};
+#[cfg(any(feature = "x11", target_os = "macos", windows))]
+use copypasta::ClipboardContext;
+use copypasta::ClipboardProvider;
```

Comment:
I find it hard sometimes to read `#[cfg(...)]`'d `use` statements like this. Would you mind grouping all the conditional imports together below the non-conditional ones? Is this a bad idea for some reason?

Reply:
Convention, we currently don't do this anywhere so the current policy is to group the imports just like any other import.

I find imports mixed in with `cfg` statements significantly more difficult to read too, but it only affects the imports that are actually annotated I'd say.

Especially in this block, the two unannotated statements `ClipboardProvider` and `NopClipboardContext` are pretty simple to understand.

So I'd probably vote against changing it, but I'd be open to other suggestions that could work around this issue? But I don't think there are any.

Reply:
Maybe better suited for a separate PR, but I could see cleaning these up. Not a big deal though, so I'll ignore it for now.